[BUG] Go build with error for SA1019: "io/ioutil" has been deprecated since Go 1.16

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->

**What happened**:

```
$ make build
...
pkg/util/system/system_file.go:21:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
	"io/ioutil"
	^
INFO File cache stats: 1 entries of total size 2.5KiB
INFO Memory: 17 samples, avg is 58.5MB, max is 95.8MB
INFO Execution took 1.559056333s
make: *** [lint-go] Error 1
```


**What you expected to happen**:

No error

**How to reproduce it (as minimally and precisely as possible)**:

```
$ make build
```

**Anything else we need to know?**:

**Environment**:
- Go version: go1.19.2 darwin/arm64

--- a/pkg/util/system/common.go
+++ b/pkg/util/system/common.go
@@ -54,8 +54,8 @@ func CommonFileWrite(file string, data string) error {
 	return os.WriteFile(file, []byte(data), 0644)
 }
 
-// ReadFileNoStat uses ioutil.ReadAll to read contents of entire file.
-// This is similar to ioutil.ReadFile but without the call to os.Stat, because
+// ReadFileNoStat uses io.ReadAll to read contents of entire file.
+// This is similar to io.ReadFile but without the call to os.Stat, because
 // many files in /proc and /sys report incorrect file sizes (either 0 or 4096).
 // Reads a max file size of 512kB.  For files larger than this, a scanner
 // should be used.

--- a/pkg/util/system/system_file.go
+++ b/pkg/util/system/system_file.go
@@ -18,7 +18,7 @@ package system
 
 import (
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
@@ -65,7 +65,7 @@ func NewProcSysctl() utilsysctl.Interface {
 }
 
 func (*ProcSysctl) GetSysctl(sysctl string) (int, error) {
-	data, err := ioutil.ReadFile(GetProcSysFilePath(sysctl))
+	data, err := os.ReadFile(GetProcSysFilePath(sysctl))
 	if err != nil {
 		return -1, err
 	}
@@ -78,7 +78,7 @@ func (*ProcSysctl) GetSysctl(sysctl string) (int, error) {
 
 // SetSysctl modifies the specified sysctl flag to the new value
 func (*ProcSysctl) SetSysctl(sysctl string, newVal int) error {
-	return ioutil.WriteFile(GetProcSysFilePath(sysctl), []byte(strconv.Itoa(newVal)), 0640)
+	return os.WriteFile(GetProcSysFilePath(sysctl), []byte(strconv.Itoa(newVal)), 0640)
 }
 
 func SetSchedGroupIdentity(enable bool) error {

